polaris-migrator/src/migrations/styles-replace-custom-property/tests/replacement-maps-basic.js

```diff
@@ -0,0 +1,9 @@
+const colorMap = {
+  '--p-text': '--p-color-text',
+  '--p-black': '#000',
+};
+
+// eslint-disable-next-line import/no-anonymous-default-export, import/no-default-export
+export default {
+  color: colorMap,
+};
```

Comment:
Could we merge this and the regex maps? I don't see a lot of value in separating them.

Reply:
Good catch! https://github.com/Shopify/polaris/pull/8265/commits/f29bd0e454b56d104ebb145c85547f1a39b23429